### PR TITLE
Add pipeline yaml for ado and fixed setup file

### PR DESCRIPTION
**Pull request by akirakakar**
## Purpose

* Updated setup.sh to create service principals and store in keyvualt
* Updated documents
* Validation and CI pipeline

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Author pre-publish checklist:

- [ ] Added test to prove my fix is effective or new feature works
- [x] No PII in logs
- [ ] Made corresponding changes to the documentation

### CODE REVIEW

**Comment by DavidBurela** [comment]
Minor thing as we get more samples in here. Maybe in the pipeline.yaml files, we should add a comment at top of the file saying if it is a Github or Azure Devops pipeline file?

File: e2e_samples/temperature_events/Infra/setup.sh [hunk 1]
```diff
@@ -48,3 +48,19 @@ az keyvault secret set --name tfstate-storage-key-dev --value $ACCOUNT_KEY_DEV -
 # Display information
 echo "Run the following command to initialize Terraform to store its state into Azure Storage:"
 echo "terraform init -backend-config=\"storage_account_name=$TF_STATE_STORAGE_ACCOUNT_NAME\"dev -backend-config=\"container_name=$TF_STATE_CONTAINER_NAME\" -backend-config=\"access_key=\$(az keyvault secret show --name tfstate-storage-key --vault-name $KEYVAULT_NAME --query value -o tsv)\" -backend-config=\"key=terraform.tfstate\""
+
+# Create Service Principal
```
Comment (DavidBurela) [inline e2e_samples/temperature_events/Infra/setup.sh:52]: This is cool

**Comment by DavidBurela** [comment]
Looks good. Demo it working in today's call and we can pull it in.

**Review by DavidBurela — APPROVED** [review]
Looks good. Show it working in today's call and we can pull it in.

File: e2e_samples/temperature_events/Infra/infra-ado-ci-pipelinet-test.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+trigger:
```
Comment (jmostella) [inline e2e_samples/temperature_events/Infra/infra-ado-ci-pipelinet-test.yml:1]: What is the difference between this one and `infra-ado-ci-pipeline.yml`? is this the renamed file?

**Review by jmostella — CHANGES_REQUESTED** [review]
I think it looks good but I think you may have a renamed file hanging around.

File: e2e_samples/temperature_events/Infra/infra-ado-ci-pipelinet-test.yml (hunk 2 above)
Comment (akirakakar) [reply e2e_samples/temperature_events/Infra/infra-ado-ci-pipelinet-test.yml:1]: infra-ado-ci-pipeline-test.yml is a validation pipeline

File: e2e_samples/temperature_events/Infra/setup.sh (hunk 1 above)
Comment (akirakakar) [reply e2e_samples/temperature_events/Infra/setup.sh:49]: Fixed minor issues we found during our testing.

**Review by jmostella — APPROVED** [review]

**Pull request merged**